django/dispatch/dispatcher.py

```diff
@@ -208,6 +211,12 @@ def send_robust(self, sender, **named):
             try:
                 response = receiver(signal=self, sender=sender, **named)
             except Exception as err:
+                logger.error(
+                    'Error calling %s in send_robust() (%s)',
```

Comment:
Whilst we can't show the signal name, I suggest using `Signal.send_robust()` in the message, to provide a little more context:

```suggestion
                    'Error calling %s in Signal.send_robust() (%s)',
```

Reply:
Added :+1: